### DISCUSSION TITLE
How to make a periodic partitioning in one or two coordinate directions?

### DISCUSSION THREAD

**Discussion by luogyong** [BrunoLevy/geogram#174] [Q&A] [answered]
Hello, Bruno,

I want to know how to make a periodic partitioning in one or two coordinate directions?

**Comment by BrunoLevy** [comment, marked as answer]
Not sure what you mean by "periodic partitioning", but as I understand, you want to select the coordinates along which the Delaunay triangulation is periodic ? It is not possible for the moment, and probably requires to dig quite deep in the implementation.

An alternative way of doing that is copying and translating the pointset along the coordinate directions that you want to make periodic. It is not completely satisfactory as it will require 9x the amount of RAM if you make it periodic along two different axes, but it will give the correct result if you take the Voronoi cells of the central zone.

**Comment by BrunoLevy** [comment]
Feature planned to be added in refactoring of Delaunay classes (#217)
